spring-integration-core/src/main/java/org/springframework/integration/config/DefaultConfiguringBeanFactoryPostProcessor.java

```diff
@@ -79,6 +81,7 @@ public class DefaultConfiguringBeanFactoryPostProcessor implements BeanDefinitio
 
 	private static final Set<Integer> REGISTRIES_PROCESSED = new HashSet<>();
 
+	@Nullable
 	private static final Class<?> XPATH_CLASS;
```

Comment:
Please, put the `@Nullable` just before the `Class<?>`.
While it works with JSpecify, but better to follow its code style recommendation.